openfe/tests/protocols/test_openmmutils.py

```diff
@@ -227,8 +227,9 @@ def get_settings():
 
 class TestFEAnalysis:
 
-    # Note: class scope _will_ cause this to segfault - the reporter has to close
-    @pytest.fixture(scope='function')
+    # Note: scope on this can sometimes cause segfault, may need to revert to
+    # function scope if it happens.
+    @pytest.fixture(scope='class')
```

Comment:
this is purely for speed-up reasons? I'm hesitant about increasing test fixture scope as a shortcut if we're at all concerned about them being mutated.

Reply:
(for the future) we discussed this and agreed that we should remove session scopes on everything.